pkg/ca/common_test.go

```diff
@@ -124,14 +124,8 @@ func TestVerifyCertChain(t *testing.T) {
 		t.Fatalf("expected error verifying cert with mismatched public keys: %v", err)
 	}
 
-	// Failure: Weak key
-	weakSubCert, weakSubKey, _ := test.GenerateWeakSubordinateCA(rootCert, rootKey)
-	err = VerifyCertChain([]*x509.Certificate{weakSubCert, rootCert}, weakSubKey)
-	if err == nil || !strings.Contains(err.Error(), "unsupported ec curve") {
-		t.Fatalf("expected error verifying weak cert chain: %v", err)
-	}
-
 	// Failure: Empty chain
+	_, weakSubKey, _ := test.GenerateWeakSubordinateCA(rootCert, rootKey)
 	err = VerifyCertChain([]*x509.Certificate{}, weakSubKey)
```

Comment:
We should update this to use `subKey` instead, but the error should be the same. (just to not reuse the weak key outside of the weak key test)

Reply:
PTAL